# Plutonium: Scrolling performance optimizations
#
# This patch tunes rasterization defaults for smoother scrolling:
# - Increase default raster thread count (can still be overridden by flag)

--- a/cc/raster/categorized_worker_pool.cc
+++ b/cc/raster/categorized_worker_pool.cc
@@ -423,7 +423,8 @@
 
   const base::CommandLine& command_line =
       *base::CommandLine::ForCurrentProcess();
-  int num_raster_threads = 1;
+  // Plutonium: use more raster threads by default for smoother scrolling.
+  int num_raster_threads = 4;
   if (command_line.HasSwitch(switches::kNumRasterThreads)) {
     std::string string_value =
         command_line.GetSwitchValueASCII(switches::kNumRasterThreads);
